templates/addons/aurora/postgresql-cf.yml

```diff
@@ -0,0 +1,153 @@
+Parameters:
```

Comment:
Why do we want to have a separated template?

Reply:
There is a discussion [here](https://github.com/aws/copilot-cli/pull/2075#discussion_r597036770) in #2075.

Basically there are a lot of places in the template that need to be rendered differently depending on whether the customer want mysql or postgresql. 

There'd be a lot of `{{if}}{{else}}{{end}}` chunks if we unify them into a single template.

Reply:
Thanks for explaining this! I didn't realize of the conversation. Still I think we should keep just one template so that we won't need to edit two almost identical templates when we modify it in the future. However, I'm also open to other ppl's idea on this.

Reply:
I will merge them into one templates since it is indeed extremely painful having to take care of two templates lol....